ceagle/main.py

```diff
@@ -44,6 +45,11 @@ def not_found(error):
     return flask.jsonify({"error": "Not Found"}), 404
 
 
+@app.errorhandler(client.UnknownService)
+def handle_unknown_service(ex):
+    return flask.jsonify({"error": str(ex)}), 400
```

Comment:
I thing 404 is a bit more appropriate. It basically means all health/security/availability/whatever is not configured and all respective urls are unavailable. While 400 is usually viewed as some sort of problem with request itself https://httpstatuses.com/400